app.js

```diff
@@ -42,6 +42,8 @@ const sequelize = require('./api/models').sequelize;
 const redirectUrls = [
   'federalist.fr.cloud.gov',
   'federalist-staging.fr.cloud.gov',
+  'federalist.18f.gov',
```

Comment:
This looks like it will cause `federalist-staging.18f.gov` and `federalist.18f.gov` to redirect to themselves. That will result in endless redirects.

Reply:
great catch ... i think this is addressed now ... pls re-review